content/news/release-20150919.md

```diff
@@ -0,0 +1,57 @@
+<div style="margin-left:40px; font-size: 0.9em;">
+<pre>
+LANDCOM 16 HQS.
+TOP SECRET
+TO: ALL FIELD COMMANDERS
+
+AN INTERCEPTED COMMUNIQUE INDICATES THE IMMEDIATE AVAILABILITY OF A
+NEW SEPTEMBER 2015 OPENRA RELEASE.  SUBSEQUENT INTERROGATION OF THE
+MESSAGE COURIER REVEALED THAT 31 AUTHORS CONTRIBUTED MORE THAN 1150
+CHANGES TOWARDS THEIR WAR EFFORT IN THE LAST THREE MONTHS.  A LARGE
+COLLECTION OF BUG FIXES AND OTHER IMPROVEMENTS ARE SUSPECTED.
+
+YOUR ORDERS ARE TO PROCEED DIRECTLY TO <a href="/download/">www.openra.net/download/</a> AND
+DOWNLOAD THE NEW RELEASE.  RENDEZVOUS WITH OTHER COMMANDERS IN-GAME
+AND DESTROY ALL ENEMY OPPOSITION.
+
+TRANSMISSION ENDS.
+</pre>
+</div>
+<hr />
+
+Our latest release includes changes / new features that implement several popular requests, including:
+
+* Support for installing assets from an English "The First Decade" DVD.
+* Adding the ability to view the map after a match completes.
+* Adding a notification when a game admin selects a map with custom rules.
+* Recording replays for single player missions.
+* Removing the "Enemy Detected" warnings from RA and D2K.
+* Fixing bottlenecks in the AI and harvesters to dramatically improve late-game performance.
+
+Other notable changes include:
+
+* Completely overhauled in-game music behaviour (don't forget to install the music from Extras &rarr; Music!).
+* Capture screenshots (saved to the support directory) using Ctrl/Cmd+P.
+* Units will no longer pursue enemies after they disappear under the shroud.
+* Improved aircraft and paradrop behaviour in RA.
+* Added missions Atreides01a and Atreides01b and FMV installation to D2K.
+
+As usual, this is only a small sampling of the [full changelog](https://github.com/OpenRA/OpenRA/wiki/Changelog) that is available on our wiki.
```

Comment:
Good idea.  We can/should set up specific redirects for http://changelog.openra.net/release-20150614 etc.